docs/reference/manifest/runtimes.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Runtimes in the manifest file
 description: ''
-ms.date: 01/24/2020
+ms.date: 02/13/2020
 localization_priority: Normal
 ---
 
 # Runtimes element
 
-This feature is in preview. Specifies the runtime of your add-in and allows custom functions and the task pane to share global data and make function calls into each other. Should follow the `<Host>` element in your manifest file.
+This feature is in preview. Specifies the runtime of your add-in and allows custom functions and the task pane to share global data and make function calls into each other. Child of the `<Host>` element in your manifest file.
```

Comment:
Would it be useful to add a "Contained in" section that lists the Host element? [Example](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/appdomains)